pkg/chunk/gcp/bigtable_index_client.go

```diff
@@ -179,7 +188,10 @@ func (s *storageClientColumnKey) BatchWrite(ctx context.Context, batch chunk.Wri
 			rowKeys = append(rowKeys, rowKey)
 			muts = append(muts, mut)
 		}
-
+		if s.writeLimiter.Allow() == false {
+			s.writeLimiter.WaitN(ctx, len(muts))
+			backoff.Wait()
```

Comment:
Why do we need a backoff and `writeLimiter.WaitN()` is not enough?

Reply:
@pracucci  We do not need back off, will remove it.